pages/Configuring/Multi-GPU.md

```diff
@@ -0,0 +1,132 @@
+
+After doing so move to the next section.
+
+# Telling Hyprland which GPU to use
+
+After determining which "card" belongs to which GPU, we now have to tell Hyprland the GPU we want to use primarily.
+
+{{< hint type=info >}}
+
+It is generally a good idea for laptops to use the integrated GPU as the primary renderer as this preserves battery life and is practically indistinguishable from using the dedicated GPU on modern systems in most cases. Hyprland can be run on integrated GPUs smoothly. The same principle applies for desktop setups with a lower and higher power rating GPUs respectively.
+
+{{< /hint >}}
+
+We can do so by using `WLR_DRM_DEVICES` variable.
+
+```ini
+
+env = WLR_DRM_DEVICES,/dev/dri/cardN
+
+```
+
+For our case, we want to use `card1` primarily and use it to render stuff. So in our case
+
+```ini
+
+env = WLR_DRM_DEVICES,/dev/dri/card1:/dev/dri/card0
+
+```
+
+Here, we tell Hyprland to set priorities. If `card1` isn't available for whatever reason, use `card0`. So if the AMD GPU isn't available, use NVIDIA. The colon is for setting priorities, essentially.
+
+After setting this, now we confirm whether the settings applied actually work or not.
+
+To test whether the NVIDIA card is rendering anything, lets check the output of `nvidia-smi`
+
+```
+Fri Jul 14 01:14:46 2023       
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 525.125.06   Driver Version: 525.125.06   CUDA Version: 1
+2.0     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  NVIDIA GeForce ...  Off  | 00000000:01:00.0 Off |                  N/A |
+| N/A   52C    P8     2W /  30W |      3MiB /  4096MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|    0   N/A  N/A      2869      G   /usr/bin/Hyprland                   1MiB |     
++-----------------------------------------------------------------------------+
+```
+
+This is with running a browser and some other stuff. We can confidentally say NVIDIA is not doing anything and is idling.
```

Comment:
I don't think this is a good way of telling whether the card runs or not, since Nvdia is blocklisted for hardware acceleration iirc. Something like `vkcube` or `glxinfo` would be more adequate.